.github/workflows/cpp.yaml

```diff
@@ -46,7 +46,7 @@ jobs:
         uses: actions/checkout@v3
         with:
           repository: ydb-platform/ydb
-          ref: 341849f5d448ac6e7362551e08f40533bbfb0ffb
+          ref: 0c915e616b63e7047189c91e523a47e6476bd3df # TODO UPDATE WITH WORKING COMMIT
```

Comment:
Let's do this before moving the link? In the current situation, new nightly images will be broken(

Reply:
Yes, I meant "DO NOT MERGE UNTIL A WORKING COMMIT IS CREATED" :)